examples/nirfmxwlan/ofdm-modacc.py

```diff
@@ -76,233 +77,232 @@
-        client.Initiate(
-            nirfmxwlan_types.InitiateRequest(instrument=instr, selector_string="", result_name="")
+
+    initiate_response = client.Initiate(
+        nirfmxwlan_types.InitiateRequest(
+            instrument=instr,
+            selector_string="",
+            result_name="",
         )
     )
+    check_for_warning(initiate_response, instr)
 
     ### Fetch Results ###
 
-    ofdm_mod_acc_fetch_composite_rmsevm_response = raise_if_error(
-        client.OFDMModAccFetchCompositeRMSEVM(
-            nirfmxwlan_types.OFDMModAccFetchCompositeRMSEVMRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+    ofdm_mod_acc_fetch_composite_rmsevm_response = client.OFDMModAccFetchCompositeRMSEVM(
+        nirfmxwlan_types.OFDMModAccFetchCompositeRMSEVMRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(ofdm_mod_acc_fetch_composite_rmsevm_response, instr)
     composite_rmsevm_mean = ofdm_mod_acc_fetch_composite_rmsevm_response.composite_rms_evm_mean
     composite_data_rmsevm_mean = (
         ofdm_mod_acc_fetch_composite_rmsevm_response.composite_data_rms_evm_mean
     )
     composite_pilot_rmsevm_mean = (
         ofdm_mod_acc_fetch_composite_rmsevm_response.composite_pilot_rms_evm_mean
     )
-    ofdm_mod_acc_fetch_numberof_symbols_used_response = raise_if_error(
-        client.OFDMModAccFetchNumberofSymbolsUsed(
-            nirfmxwlan_types.OFDMModAccFetchNumberofSymbolsUsedRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+
+    ofdm_mod_acc_fetch_numberof_symbols_used_response = client.OFDMModAccFetchNumberofSymbolsUsed(
+        nirfmxwlan_types.OFDMModAccFetchNumberofSymbolsUsedRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(ofdm_mod_acc_fetch_numberof_symbols_used_response, instr)
     number_of_symbols_used = (
         ofdm_mod_acc_fetch_numberof_symbols_used_response.number_of_symbols_used
     )
-    ofdm_mod_acc_fetch_frequency_error_mean_response = raise_if_error(
-        client.OFDMModAccFetchFrequencyErrorMean(
-            nirfmxwlan_types.OFDMModAccFetchFrequencyErrorMeanRequest(
-                instrument=instr, selector_string="", timeout=10.0
-            )
+
+    ofdm_mod_acc_fetch_frequency_error_mean_response = client.OFDMModAccFetchFrequencyErrorMean(
+        nirfmxwlan_types.OFDMModAccFetchFrequencyErrorMeanRequest(
+            instrument=instr,
+            selector_string="",
+            timeout=10.0,
         )
     )
+    check_for_warning(ofdm_mod_acc_fetch_frequency_error_mean_response, instr)
     frequency_error_mean = ofdm_mod_acc_fetch_frequency_error_mean_response.frequency_error_mean
-    ofdm_mod_acc_fetch_symbol_clock_error_mean_response = raise_if_error(
+
+    ofdm_mod_acc_fetch_symbol_clock_error_mean_response = (
```

Comment:
Another inexplicably long name. This is the OFDM-MODACC example. Therefore, I suggest removing any `ofdm_mod_acc_` prefix from variable names in this example.

Reply:
removed these prefixes